conftest.py

```diff
@@ -2,9 +2,10 @@
 import pytest
 
 
+# Shared across Python and docs tests
 @pytest.fixture(scope="session")
 def spark():
-    sess = SparkSession.builder\
+    sess = SparkSession.builder \
         .master("local[2]") \
         .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer") \
```

Comment:
Can we remove this now?

Reply:
As discussed offline, we still use Kryo serializers in all of our tests (even if we don't require registration).